Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/DriveBase.java

```diff
@@ -32,23 +33,30 @@
 	public DriveBase() {
 		super();
 
-		// leftFrontMotor = new WPI_TalonSRX(RobotMap.LEFT_FRONT);
-		// leftBackMotor = new WPI_TalonSRX(RobotMap.LEFT_BACK);
-		// rightFrontMotor = new WPI_TalonSRX(RobotMap.RIGHT_FRONT);
-		// rightBackMotor = new WPI_TalonSRX(RobotMap.RIGHT_BACK);
+		// TODO: This should be set back... the other code (lines 42 and 43 was test
+		// code)
+		// TODO: make sure to look at the ReferenceDocuments/Talon SRX Victor SPX -
+		// Software Reference Manual.pdf
+		// Section 7.6. Multiple Talon SRXs and single sensor
+		// FAILURE TO DO SO MAY BURN OUT THE MOTORS!!!!!
 
-		leftFrontMotor = new WPI_TalonSRX(0);
-		rightFrontMotor = new WPI_TalonSRX(1);
+		leftFrontMotor = new WPI_TalonSRX(RobotMap.LEFT_FRONT);
+		leftBackMotor = new WPI_TalonSRX(RobotMap.LEFT_BACK);
+		rightFrontMotor = new WPI_TalonSRX(RobotMap.RIGHT_FRONT);
+		rightBackMotor = new WPI_TalonSRX(RobotMap.RIGHT_BACK);
 
-		// leftBackMotor.set(ControlMode.Follower, RobotMap.LEFT_FRONT);
-		// rightBackMotor.set(ControlMode.Follower, RobotMap.RIGHT_FRONT);
+		// leftFrontMotor = new WPI_TalonSRX(0);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)